packages/wallet/backend/src/app.ts

```diff
@@ -179,6 +179,11 @@ export class App {
 
     // outgoing payment routes
     router.post('/outgoing-payments', isAuth, outgoingPaymentController.create)
+    router.post(
```

Comment:
I think a better separation of endpoints will be
1. `POST /quotes` - as at the first request a quote is created and not an outgoing payments
2. `POST /outgoing-payments` - same endpoint that exists now, just that will get the quote as a required param